lib/client.js

```diff
@@ -300,6 +300,7 @@ function prepareConfig({
     return kafkaConfig
   }
 
+  /* istanbul ignore next */
```

Comment:
Because the only way to reach that code at test time is to start a Kafka instance that requires username and password.  If it might not be useful, it can be removed without any problem.